student_explorer/templates/403.html

```diff
@@ -7,9 +7,17 @@
 {% block content %}
 <div class="container">
     <h1>Permission Denied</h1>
-    <p>You do not currently have access to Student Explorer. To learn about Student Explorer and its history, see 
-        <a href="/about">About Student Explorer</a>. For more information about using Student Explorer, including 
-        how to get access and who to contact for assistance, see 
-        <a href="https://documentation.its.umich.edu/student-explorer-general">Student Explorer Resources & Support.</a></p>
+    <p>You do not currently have access to Student Explorer.</p>
+    <h2 class="list-header">What is Student Explorer?</h2>
+    <p>To learn about Student Explorer and its history, see 
+        <a href="/about">About Student Explorer</a>. For more information 
+        about using Student Explorer, see 
+        <a href="https://documentation.its.umich.edu/student-explorer-general">Student 
+            Explorer Resources & Support.</a></p>
+    <h2 class="list-header">How do I request access to Student Explorer?</h2>
+    <p><a href="https://documentation.its.umich.edu/student-explorer-access">Learn how access to Student Explorer is governed,
+        who can get access to it, and who already has access</a> or go ahead and
```

Comment:
Maybe add a comma before "or go ahead ...", since it's a new independent clause? You could also probably remove "go ahead"; it seems a little wordy to me.

Reply:
My rationale was just trying to create a clear visual separation between the two links so it's clear they are two links and not just one very long link. However I am open to being convinced otherwise.